enterprise/frontend/src/embedding-sdk/components/private/SdkLicenseProblemBanner/SdkLicenseProblemBanner.module.css

```diff
@@ -0,0 +1,34 @@
+.PopoverDropdown.PopoverDropdown {
+  border: none;
+}
+
+.Indicator {
+  cursor: pointer;
+  border-radius: 4px;
+  box-shadow: 0 6px 24px 0 #0000001a;
```

Comment:
The shadow in the Figma design differs by what we have in our `Paper` component. This is fine as it needs to be visually distinctive on top of the user's application.